### PR TITLE
Increase severity of some rules and fix instances

### PR DESCRIPTION
**Pull request by Evangelink**
@MarcoRossignoli Please let me know if there are parts of the "default" state you would like to tweak.

### CODE REVIEW

File: src/coverlet.collector/Utilities/TestPlatformEqtTrace.cs [hunk 1]
```diff
@@ -8,17 +8,17 @@ namespace Coverlet.Collector.Utilities
     /// <summary>
     /// Test platform eqttrace
     /// </summary>
-    internal class TestPlatformEqtTrace
+    internal static class TestPlatformEqtTrace
```
Comment (Evangelink) [inline src/coverlet.collector/Utilities/TestPlatformEqtTrace.cs:11]: Updated the type to be `static` as all members could be `static`.  If you prefer an instance with the idea of one day have a different instance for tests/prod then let me know I can revert. We could also wait for such need and refactor and that time.

File: src/coverlet.core/CoverageSummary.cs [hunk 2]
```diff
@@ -7,17 +7,17 @@
 
 namespace Coverlet.Core
 {
-    internal class CoverageSummary
+    internal static class CoverageSummary
```
Comment (Evangelink) [inline src/coverlet.core/CoverageSummary.cs:10]: Same remark as for the eqt logger. All members could be put `static` so I have updated everything.

File: src/coverlet.console/Logging/ConsoleLogger.cs [hunk 3]
```diff
@@ -9,7 +9,8 @@ namespace Coverlet.Console.Logging
 {
     class ConsoleLogger : ILogger
     {
-        private static readonly object _sync = new object();
+        private static readonly object s_sync = new();
```
Comment (Evangelink) [inline src/coverlet.console/Logging/ConsoleLogger.cs:12]: `s_` is the default that was approved but if you want to change it to make no diff or use a different pattern, let me know.

File: src/coverlet.core/Instrumentation/ReachabilityHelper.cs [hunk 4]
```diff
@@ -169,7 +169,9 @@ public override string ToString()
         /// OpCodes that transfer control code, even if they do not
         /// introduce branch points.
         /// </summary>
+#pragma warning disable IDE1006 // Naming Styles
```
Comment (Evangelink) [inline src/coverlet.core/Instrumentation/ReachabilityHelper.cs:172]: I thought you would prefer not to change the name of the field but if you are ok with changing, please let me know.

File: src/coverlet.core/Instrumentation/ReachabilityHelper.cs [hunk 5]
```diff
@@ -224,7 +226,9 @@ public override string ToString()
         /// OpCodes that unconditionally transfer control, so there
         /// is not "fall through" branch target.
         /// </summary>
+#pragma warning disable IDE1006 // Naming Styles
```
Comment (Evangelink) [inline src/coverlet.core/Instrumentation/ReachabilityHelper.cs:229]: I thought you would prefer not to change the name of the field but if you are ok with changing, please let me know.

File: src/coverlet.collector/Utilities/TestPlatformEqtTrace.cs (hunk 1 above)
Comment (MarcoRossignoli) [reply src/coverlet.collector/Utilities/TestPlatformEqtTrace.cs:11]: Yep pls revert I want to able to test/inject everything

File: src/coverlet.core/CoverageSummary.cs (hunk 2 above)
Comment (MarcoRossignoli) [reply src/coverlet.core/CoverageSummary.cs:10]: the same

File: src/coverlet.core/Instrumentation/ReachabilityHelper.cs (hunk 4 above)
Comment (MarcoRossignoli) [reply src/coverlet.core/Instrumentation/ReachabilityHelper.cs:172]: I'm ok with field name update

File: src/coverlet.core/Instrumentation/ReachabilityHelper.cs (hunk 5 above)
Comment (MarcoRossignoli) [reply src/coverlet.core/Instrumentation/ReachabilityHelper.cs:229]: I'm ok with it

File: src/coverlet.console/Logging/ConsoleLogger.cs (hunk 3 above)
Comment (MarcoRossignoli) [reply src/coverlet.console/Logging/ConsoleLogger.cs:12]: I'm ok with field name update

**Comment by MarcoRossignoli** [comment]
@Evangelink pls keep the instances(no static) and I'm ok with the `s_` renaming

**Review by MarcoRossignoli — APPROVED** [review]
thx!

**Pull request merged**